app/upgrades/v20/upgrades.go

```diff
@@ -43,3 +51,39 @@ func EnableGovPrecompile(ctx sdk.Context, ek *evmkeeper.Keeper) error {
 	}
 	return ek.SetParams(ctx, params)
 }
+
+func UpdateExpeditedPropsParams(ctx sdk.Context, gv govkeeper.Keeper) error {
+	params, err := gv.Params.Get(ctx)
+	if err != nil {
+		return err
+	}
+
+	// use the same denom as the min deposit denom
+	// !! NOTE: for mainnet upgrade updating the var sdk.DefaultBondDenom = 'aevmos'
+	// !! before running the migrations will suffice to set the expedited
+	// !! min deposit denom
+	// also amount must be greater than MinDeposit amount
+	// !! NOTE for mainnet upgrade, updating the var govv1.DefaultMinDepositTokens
+	// !! before running the migrations
+	// !! to the current MinDeposit value will set the MinExpDeposit to be 5x this value
+	denom := params.MinDeposit[0].Denom
+	expDepAmt := params.ExpeditedMinDeposit[0].Amount
+	if expDepAmt.LTE(params.MinDeposit[0].Amount) {
+		expDepAmt = params.MinDeposit[0].Amount.MulRaw(govv1.DefaultMinExpeditedDepositTokensRatio)
+	}
+	params.ExpeditedMinDeposit = sdk.NewCoins(sdk.NewCoin(denom, expDepAmt))
+
+	// if expedited voting period > voting period
+	// set expedited voting period to be half the voting period
+	// !! NOTE this is needed only on testnet
+	// !! on mainnet we can have the default value (1 day)
+	if params.ExpeditedVotingPeriod != nil && params.VotingPeriod != nil && *params.ExpeditedVotingPeriod > *params.VotingPeriod {
```

Comment:
Use the isTestnet util

Reply:
will be a no-op as it is on mainnet anyways, because `*params.ExpeditedVotingPeriod < *params.VotingPeriod`